### ISSUE TITLE
Mod Crash

### ISSUE THREAD

**Issue by cgc007** [Suprcheese/Orbital-Ion-Cannon#29]
The following occurs if auto-targeting is off:

```
The mod Orbital Ion Cannon caused a non-recoverable error.
Please report this error to the mod author.

Error while running event Orbital Ion Cannon::on_biter_base_built (ID 55)
__Orbital Ion Cannon__/autotargeter.lua:65: attempt to index global 'Chunk' (a nil value)
stack traceback:
	__Orbital Ion Cannon__/autotargeter.lua:65: in function <__Orbital Ion Cannon__/autotargeter.lua:61>

```
Also happened previously with the following info. This occurs if autotargeting is checked.
```
Error while running event Orbital Ion Cannon::on_sector_scanned (ID 7)
__Orbital Ion Cannon__/autotargeter.lua:2: attempt to index global 'Chunk' (a nil value)
stack traceback:
	__Orbital Ion Cannon__/autotargeter.lua:2: in function 'findNestNear'
	__Orbital Ion Cannon__/autotargeter.lua:46: in function <__Orbital Ion Cannon__/autotargeter.lua:16>
```

Seems to be tied to watching a new base be built under radar coverage, or when a base is discovered by radar scanning.

Both are reproducible under my save. I cannot play for more than about 15 seconds.

Was working fine before this most recent update for me.

**Comment by cgc007** [comment]
I did some more research. I removed the mod from my game. Loaded the save to remove all ion cannons. Re-enabled the mod. Launched 4 ion cannons (same number I had previously) with the cheat panel. Until I enabled the `auto-targeting` research, no errors occurred. As soon as I did (with all 4 ions ready to fire) the game crashed.

If I leave auto-targeting unresearched and fire the cannons manually, it also works fine.

HTH.
